applications/luci-app-tailscale-community/htdocs/luci-static/resources/view/tailscale.js

```diff
@@ -0,0 +1,356 @@
+    } else {
+        // Define headers for the peers table.
+        const peerTableHeaders = [
+            { text: _('Status'), style: 'width: 80px;' },
+            { text: _('Hostname') },
+            { text: _('Tailscale IP') },
+            { text: _('OS') },
+            { text: _('Connection Info') },
+            { text: _('RX') },
+            { text: _('TX') }
+        ];
+        
+        // Build the peers table.
+        peersContent = E('table', { 'class': 'cbi-table' }, [
+            // Table Header Row
+            E('tr', { 'class': 'cbi-table-header' }, peerTableHeaders.map(header => {
+                let th_style = 'padding-right: 20px; text-align: left;';
+                if (header.style) {
+                    th_style += header.style;
+                }
+                return E('th', { 'class': 'cbi-table-cell', 'style': th_style }, header.text);
+            })),
+            
+            // Table Body Rows (one for each peer)
+            ...Object.entries(peers).map(([hostname, peer]) => {
+                const isOnline = peer.status !== 'offline';
+                const td_style = 'padding-right: 20px;';
+
+                return E('tr', { 'class': 'cbi-rowstyle-1' }, [
+                    E('td', { 'class': 'cbi-value-field', 'style': td_style },
+                        E('span', {
+                            'style': `color:${isOnline ? 'green' : 'gray'};`,
+                            'title': isOnline ? _('Online') : _('Offline')
+                        }, isOnline ? '●' : '○')
+                    ),
+                    E('td', { 'class': 'cbi-value-field', 'style': td_style }, E('strong', {}, hostname)),
+                    E('td', { 'class': 'cbi-value-field', 'style': td_style }, peer.ip || 'N/A'),
+                    E('td', { 'class': 'cbi-value-field', 'style': td_style }, peer.ostype || 'N/A'),
+                    E('td', { 'class': 'cbi-value-field', 'style': td_style }, peer.linkadress || '-'),
+                    E('td', { 'class': 'cbi-value-field', 'style': td_style }, formatBytes(peer.rx)),
+                    E('td', { 'class': 'cbi-value-field', 'style': td_style }, formatBytes(peer.tx))
+                ]);
+            })
+        ]);
+    }
+
+    // Combine all parts into a single DocumentFragment.
+    // Using E() without a tag name creates a fragment, which is perfect for grouping elements.
+    return E([
+        statusTable,
+        E('div', { 'style': 'margin-top: 25px;' }, [
+            E('h4', {}, _('Network Devices')),
+            peersContent
+        ])
+    ]);
+}
+
+return view.extend({
+    load: function() {
+        return Promise.all([
+            L.resolveDefault(callGetStatus(), { running: '', peers: [] }),
+            L.resolveDefault(callGetSettings(), { accept_routes: false }),
+            L.resolveDefault(callGetSubroutes(), { routes: [] })
+        ])
+        .then(function(rpc_data) {

```

Comment:
`        .then(function({status, settings, routes}) {`
and unpack as necessary further down as objects or arrays.